counterpartylib/lib/blocks.py

```diff
@@ -326,18 +326,27 @@ def initialise(db):
     cursor.execute('''CREATE TABLE IF NOT EXISTS assets(
                       asset_id TEXT UNIQUE,
                       asset_name TEXT UNIQUE,
-                      block_index INTEGER)
+                      block_index INTEGER,
+                      asset_longname TEXT)
                    ''')
     cursor.execute('''CREATE INDEX IF NOT EXISTS
                       name_idx ON assets (asset_name)
                    ''')
     cursor.execute('''CREATE INDEX IF NOT EXISTS
                       id_idx ON assets (asset_id)
                    ''')
+
+    # Add asset_longname for sub-assets
+    #   SQLite can’t do `ALTER TABLE IF COLUMN NOT EXISTS`.
+    columns = [column['name'] for column in cursor.execute('''PRAGMA table_info(assets)''')]
+    if 'asset_longname' not in columns:
+        cursor.execute('''ALTER TABLE assets ADD COLUMN asset_longname TEXT''')
+    cursor.execute('''CREATE UNIQUE INDEX IF NOT EXISTS asset_longname_idx ON assets(asset_longname)''')
```

Comment:
So, if this is the first installation. The table gets created with an asset_longname column, but if this is an upgrade and the table does not contain an asset_longname column, add it. Got it.

My comment would be around failures to add the column. I suppose at around 60-70k rows it's not that much of an issue, but I'm just going through and calling out what I see. Don't take my review too seriously.